packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -327,18 +332,27 @@ class AndroidCameraCameraX extends CameraPlatform {
     CameraDescription cameraDescription,
     MediaSettings? mediaSettings,
   ) async {
+    // TODO(bparrishMines): Failure to get permissions isn't actually handled.
     // Must obtain proper permissions before attempting to access a camera.
-    await proxy.requestCameraPermissions(mediaSettings?.enableAudio ?? false);
+    await systemServicesManager.requestCameraPermissions(
+      mediaSettings?.enableAudio ?? false,
+    );
 
     // Save CameraSelector that matches cameraDescription.
-    final int cameraSelectorLensDirection =
+    final LensFacing cameraSelectorLensDirection =
         _getCameraSelectorLensDirection(cameraDescription.lensDirection);
-    cameraIsFrontFacing =
-        cameraSelectorLensDirection == CameraSelector.lensFacingFront;
-    cameraSelector = proxy.createCameraSelector(cameraSelectorLensDirection);
+    cameraIsFrontFacing = cameraSelectorLensDirection == LensFacing.front;
+    cameraSelector = proxy.newCameraSelector(
+      requireLensFacing: cameraSelectorLensDirection,
+    );
     // Start listening for device orientation changes preceding camera creation.
-    proxy.startListeningForDeviceOrientationChange(
-        cameraIsFrontFacing, cameraDescription.sensorOrientation);
+    // TODO(bparrishMines): `isFrontFacing` and `sensorOrientation` aren't used
+    // by the DeviceOrientationManager and can be removed if this was
+    // intentional.
```

Comment:
Yeah this can be removed. It was a copy/paste sorta error on my part. Thanks for catching this!